addons/membership/security/ir.model.access.csv

```diff
@@ -1,5 +1,6 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-access_membership_membership_line_partner_manager,membership.membership_line partner_manager,model_membership_membership_line,base.group_partner_manager,1,1,1,1
-access_membership_membership_line_employee,membership.membership_line,model_membership_membership_line,base.group_user,1,0,0,0
-access_report_membership,report.membership,model_report_membership,base.group_partner_manager,1,0,0,0
-access_membership_invoice,access.membership.invoice,model_membership_invoice,account.group_account_invoice,1,1,1,0
+access_res_partner_grade,res.partner.grade,model_res_partner_grade,sales_team.group_sale_salesman,1,1,1,0
+access_res_partner_grade_employee,res.partner.grade,model_res_partner_grade,base.group_user,1,0,0,0
+access_res_partner_grade_portal,res.partner.grade,model_res_partner_grade,base.group_portal,1,0,0,0

```

Comment:
Why should portal / public have access to grades ? Is it really required from data point of view ?

Reply:
True, I left it in the website_crm_partner_assign module